security/container_content.adoc

```diff
@@ -0,0 +1,53 @@
+[[security-content]]
+= Container Content
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-content-whats-in-your-container]]
+== What's In Your Container?
+
+When it comes to security, what is inside your container matters. For some time
+now, applications and infrastructures have been composed from readily available
```

Comment:
s/For some time no, applications and infrastructures have been composed from/Applications and infrastructures are composed of